### DISCUSSION TITLE
ocean daily files link to COMROT

### DISCUSSION THREAD

**Discussion by jiandewang** [NOAA-EMC/global-workflow#2554] [General]
during HR3 runs I noticed that the ocean daily files in DATA dir that are linked to COMROT is not totally correct. Let me use IC=20191206  7 day run as an example to explain:

for IC=20191206 we should have ocean daily files ocean_daily_2019_12_06.nc,  ...... until ocean_daily_2019_12_12.nc. Note here ocean_daily_2019_12_06.nc is the daily average for the first 24hr.

However in https://github.com/NOAA-EMC/global-workflow/blob/develop/ush/forecast_postdet.sh#L439-L444, it will give us the follwoing in the DATA dir:
-rw-r--r-- 1 jiande.wang emc 105894908 Apr 26 03:29 ocn_daily_2019_12_06.nc
lrwxrwxrwx 1 jiande.wang emc       153 Apr 26 02:46 ocn_daily_2019_12_07.nc -> /lfs/h2/emc/ptmp/jiande.wang/HR3-work/HR3-20191206/COMROOT/2019120600/HR3-20191206/gfs.20191206/00//model_data/ocean/history/gfs.ocean.t00z.daily.f02
4.nc
lrwxrwxrwx 1 jiande.wang emc       153 Apr 26 02:46 ocn_daily_2019_12_08.nc -> /lfs/h2/emc/ptmp/jiande.wang/HR3-work/HR3-20191206/COMROOT/2019120600/HR3-20191206/gfs.20191206/00//model_data/ocean/history/gfs.ocean.t00z.daily.f04
8.nc
lrwxrwxrwx 1 jiande.wang emc       153 Apr 26 02:46 ocn_daily_2019_12_09.nc -> /lfs/h2/emc/ptmp/jiande.wang/HR3-work/HR3-20191206/COMROOT/2019120600/HR3-20191206/gfs.20191206/00//model_data/ocean/history/gfs.ocean.t00z.daily.f07
2.nc
lrwxrwxrwx 1 jiande.wang emc       153 Apr 26 02:46 ocn_daily_2019_12_10.nc -> /lfs/h2/emc/ptmp/jiande.wang/HR3-work/HR3-20191206/COMROOT/2019120600/HR3-20191206/gfs.20191206/00//model_data/ocean/history/gfs.ocean.t00z.daily.f09
6.nc
lrwxrwxrwx 1 jiande.wang emc       153 Apr 26 02:46 ocn_daily_2019_12_11.nc -> /lfs/h2/emc/ptmp/jiande.wang/HR3-work/HR3-20191206/COMROOT/2019120600/HR3-20191206/gfs.20191206/00//model_data/ocean/history/gfs.ocean.t00z.daily.f12
0.nc
lrwxrwxrwx 1 jiande.wang emc       153 Apr 26 02:46 ocn_daily_2019_12_12.nc -> /lfs/h2/emc/ptmp/jiande.wang/HR3-work/HR3-20191206/COMROOT/2019120600/HR3-20191206/gfs.20191206/00//model_data/ocean/history/gfs.ocean.t00z.daily.f14
4.nc
lrwxrwxrwx 1 jiande.wang emc       153 Apr 26 02:46 ocn_daily_2019_12_13.nc -> /lfs/h2/emc/ptmp/jiande.wang/HR3-work/HR3-20191206/COMROOT/2019120600/HR3-20191206/gfs.20191206/00//model_data/ocean/history/gfs.ocean.t00z.daily.f16
8.nc

so day 1 averaged file (ocn_daily_2019_12_06.nc) isn't being linked to COMROT while there is an unnecessary "ocn_daily_2019_12_13.nc" file linked to the last day (7th) of the run. Note for IC=20191206 7 day run the last day should be 2019_12_12.
All this means is that there is a 24hr shift while doing the link.

**Comment by aerorahul** [comment]
This issue was resolved with PR #2689
